app/modules/sightings/models.py

```diff
@@ -1059,7 +1059,9 @@ def _get_annotation_id_data_from_job(self, t_annot_result, q_annot_job):
     # Helper to ensure that the required annot and individual data is present
     def _ensure_annot_data_in_response(self, annot, response):
 
+        # will populate individual_first_name in next block to save a database hit
         individual_guid = annot.encounter.individual_guid if annot.encounter else None
+        individual = Individual.query.get(individual_guid) if individual_guid else None
 
         if annot.guid not in response['annotation_data'].keys():
             encounter_location = (
```

Comment:
could also use `annot.get_location_id()` here -- but/and this has the caveat of using the _Sighting_ `locationId` as a fallback if the Encounter does not have one.  would this be a feature here? or a bug?  (as this property is called `encounter_location` specifically.  **important**: this Sighting-fallback behavior happens on `enc.get_location_id()` as well!  so its already happening.)   note: you can also override this behavior with `get_location_id(sighting_fallback=False)`.

Reply:
Per convo in slack, I think using the sighting locationId is best here.